### ISSUE TITLE
compiler-builtins emits a lof of warnings about conflicts method name

### ISSUE THREAD

**Issue by JohnTitor** [rust-lang/compiler-builtins#391]
We can see a lot of warnings when building `compiler-builtins`:
```
 warning: a method with this name may be added to the standard library in the future
   --> /home/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/compiler_builtins-0.1.35/src/int/udiv.rs:98:23
    |
98  |             q = n << (<$ty>::BITS - sr);
    |                       ^^^^^^^^^^^
...
268 |         udivmod_inner!(n, d, rem, u128)
    |         ------------------------------- in this macro invocation
    |
    = warning: once this method is added to the standard library, the ambiguity may cause an error or change in behavior!
    = note: for more information, see issue #48919 <rust-lang/rust/issues/48919>
    = help: call with fully qualified syntax `Int::BITS(...)` to keep using the current method
    = help: add `#![feature(int_bits_const)]` to the crate attributes to enable `num::<impl u128>::BITS`
    = note: this warning originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)
```

The problem is the same name item is added by https://github.com/rust-lang/rust/pull/76492. It'd be great if we could tweak (rename or call them more specifically) code here to remove this warning :)

**Comment by AaronKutch** [comment]
I have fixed this in PR #384, just need to get it merged

**Comment by AaronKutch** [comment]
fixed on master now

**Comment by JohnTitor** [comment]
Awesome!

@Amanieu Would you mind making a new release?

**Comment by Amanieu** [comment]
Done.
